.gitignore

```
@@ -40,6 +40,7 @@ dump.rdb
 .transifexrc
 python_env
 python_env-3.6
+python_env-3.9
```

Should we make this a wildcard so it's more future-proof?

```suggestion
python_env-*
```